website/scripts/build-static-content.js

```diff
@@ -512,6 +512,135 @@ module.exports = {
           }//∞ </each source file>
         }//∞ </each section repo path>
 
+        // After we build the Markdown pages, we'll merge the osquery schema with the Fleet schema overrides, then create EJS partials for each table in the merged schema.
+
+        let urlOfOsquerySchemaJson = 'https://raw.githubusercontent.com/osquery/osquery-site/source/src/data/osquery_schema_versions/5.4.0.json';
+        let rawOsqueryTables = await sails.helpers.http.get(urlOfOsquerySchemaJson);
+        let fleetOverridesForTables = await sails.helpers.fs.readJson(path.resolve(topLvlRepoPath+'/schema', 'fleet_schema.json'));
+
+        let expandedTables = []; // create an empty array for the merged schema.
+
+        for(let osquerySchemaTable of rawOsqueryTables) {
+          let expandedTableToPush = osquerySchemaTable; // Set the initial value for the table in the merged JSON
```

Comment:
@mikermcneil I was setting the initial value of `expandedTableToPush` to be the `osquerySchemaTable` because the code below only adds tables to the merged schema if they have overrides from the Fleet-specific schema. 

If we update this to set it to an empty object initially, we would still need to set it to be the `osquerySchemaTable` before we add it to the final schema. What are your thoughts?

Reply:
Let me rephrase my feedback to be less specific and give you some more space to maneuver, sorry:
1. Remove deep merging, it can't be trusted
2. Make sure you aren't mutating the original.  Remember, any array or dictionary (at any level of depth) can be mutated if you don't rebuild it (e.g. shallow clone first before changing its items/keys).

Reply:
@eashaw Let me rephrase my feedback to be less specific and give you some more space to maneuver, sorry:
1. Remove deep merging, it can't be trusted
2. Make sure you aren't mutating the original.  Remember, any array or dictionary (at any level of depth) can be mutated if you don't rebuild it (e.g. shallow clone first before changing its items/keys).